tests/scripts/expect/cli-scan-discover.exp

```diff
@@ -53,7 +53,7 @@ set extpan $expect_out(1,string)
 expect "Done"
 send "networkname\n"
 expect "networkname"
-expect -re {[\r\n]([^\r\n]+?)[\r\n]}
+expect -re {[^\w-]([\w-]+)[^\w-]}
```

Comment:
network name can contain non-word characters, such as spaces.

Reply:
Changed to another approach.